data/themes/darktable.css

```diff
@@ -1539,8 +1539,9 @@ margin: 0 2px;
 {
   min-height: 0.95em;
   min-width: 0.95em;
-  margin: 0 0.75em;
+  margin: 0 0.45em;
   border: none;
+  padding: 0.3em;
```

Comment:
`padding: 0.2em 0.3em 0.3em 0.2em;`

Reply:
fine with this - it does indeed make the borders more even.